x/auth/types/credentials_test.go

```diff
@@ -41,17 +40,25 @@ func TestNewModuleCrendentials(t *testing.T) {
 	c, err = authtypes.NewModuleCredential("group", []byte{0x20})
 	require.NoError(t, err)
 	require.False(t, credential.Equals(c))
+
+	address := secp256k1.GenPrivKey().PubKey().Address()
+	expected := sdk.MustAccAddressFromHex(address.String())
+	c, err = authtypes.NewModuleCredential("group", address)
+	require.NoError(t, err)
+	require.Equal(t, strings.ToLower(expected.String()), "0x"+strings.ToLower(address.String()))
```

Comment:
Hmm, so we are deliberately ignoring the assertion for address calculated from the credential ?

Reply:
Yeah until this tasked is picked up https://polygon.atlassian.net/browse/POS-2493
But let me mark the code with a `TODO`